test/WebJobs.Script.Tests.Integration/TestScripts/Proxies/proxies.json

```diff
@@ -199,11 +199,20 @@
         "response.headers.Content-Length": "888",
         "response.headers.Test": "test"
       }
-
+    },
+    "pingMakeResponseProxy": {
+      "matchCondition": {
+        "route": "pingMakeResponseProxy"
+      },
+      "backendUri": "http://localhost/api/pingMakeResponse",
+      "requestOverrides": {
+        "backend.request.headers.accept": "text/plain"
+      }
```

Comment:
Can you add a responseOverrides section here as well to make sure they are also coming back along with response headers from Function itself?

Reply:
Done